tests/protocols/test_websocket.py

```diff
@@ -692,7 +690,7 @@ async def get_subprotocol(url: str):
 
 
 MAX_WS_BYTES = 1024 * 1024 * 16
-MAX_WS_BYTES_PLUS1 = MAX_WS_BYTES + 1
+MAX_WS_BYTES_PLUS1 = MAX_WS_BYTES + 10
```

Comment:
@aaugustin Do you know why I had to add 10 instead of 1 here? 🤔 Did something changed on the legacy server protocol in this regard?